turbopack/crates/turbopack-browser/src/ecmascript/content.rs

```diff
@@ -128,7 +128,7 @@ impl EcmascriptBrowserChunkContent {
         let mut code = code.build();
 
         if let MinifyType::Minify { mangle } = this.chunking_context.await?.minify_type() {
-            code = minify(&code, source_maps, mangle)?;
+            code = minify(&code, source_maps, false, mangle)?;
```

Comment:
This code path is also used for `edge` code, which is also server side code.

You probably need a property in the browser chunking context to control that.
Similar to `current_chunk_method`